egs2/kising/svs1/local/midi-note.scp

```diff
@@ -0,0 +1,108 @@
+127	G9
```

Comment:
You can do the symlink to other recipes to reduce unnecessary files, since it is quite common